api/compat/v3/models.md

```diff
@@ -0,0 +1,92 @@
+# /brands/{brandId}/models
+
+Get the models list for a *brand Id*
+
+## Basics
+
+`[GET] https://cloud.xee.com/v3/compat/brands/{brandId}/models`
+
+Secured by **Basic** auth.
+
+## Request
+
+### Headers
+
+|Header name|Header value|Mandatory|
+|---|---|---|
+|Authorization|`Basic` with the `base64` of your `clientId` and  `clientSecret `|YES|
+
+### Url Parameters
+
+|Parameter name|Parameter value|Mandatory|
+|---|---|---|
+|`brandId`|The brand id field|YES|
+
+## Success Response
+
+- Status Code: `200`
+- Body:
+
+```javascript
+[
+    {
+        "brandId": 467,
```

Comment:
The purpose was to make the usage of the `model` model smooth by giving all the informations about the model in the model object